cli/internal/cmd/verify.go

```diff
@@ -180,12 +196,11 @@ func (c *verifyCmd) parseVerifyFlags(cmd *cobra.Command, fileHandler file.Handle
 	}
 	c.log.Debugf("Flag 'force' set to %t", force)
 
-	raw, err := cmd.Flags().GetBool("raw")
+	c.log.Debugf("Flag 'raw' set to %t", force)
```

Comment:
```suggestion
	c.log.Debugf("Flag 'output' set to %t", output)
```

and move to after the error check.